magstock/templates/regextra.html

```diff
@@ -1,17 +1,24 @@
 
 <script type="text/javascript">
     if (window.BADGE_TYPES) {
-        BADGE_TYPES.options[0].description = 'Includes three day admission and campground fees';
-        if (BADGE_TYPES.options[2]) {
-            BADGE_TYPES.options[2].description = 'T-shirt level plus additional swag. (Available until {{ c.SUPPORTER_DEADLINE|datetime:'%b %d' }}. Limited quantities.)';
+        for (var i = 0; i < _(BADGE_TYPES.options).size(); i++) {
+            switch (BADGE_TYPES.options[i].extra) {
+                case 0:
+                    BADGE_TYPES.options[i].description = 'Includes three day admission and campground fees.';
+                    break;
+                case {{ c.SHIRT_LEVEL }}:
+                    BADGE_TYPES.options[i] = {
+                        title: 'Add a T-Shirt',
+                        description: 'Attendee level plus a MAGStock-themed t-shirt.',
+                        extra: {{ c.SHIRT_LEVEL }}
+                    };
+                    break;
+                case {{ c.SUPPORTER_LEVEL }}:
+                    BADGE_TYPES.options[i].description = 'T-shirt level plus additional swag. (Available until {{ c.SUPPORTER_DEADLINE|datetime:'%b %d' }}. Limited quantities.)';
+                    break;
+            }
         }
-        {% if c.SHIRT_LEVEL in c.PREREG_DONATION_TIERS %}
```

Comment:
Originally, this check was for _adding_ a badge button. Since last year, however, we added the t-shirt badge button in the main repo (at https://github.com/magfest/ubersystem/blob/master/uber/templates/preregistration/preregbase.html#L80 specifically) so the check for the shirt level existing is no longer needed - as long as the button exists, we know it's allowed to exist.